backend/Helpers/SafeTransportMethods.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using OpenData.Domain.Models;
+
+namespace OpenData.Helpers
+{
+    public static class SafeTransportMethods
```

Comment:
We can use the automapper for this instead

Reply:
Resolved in newer commit.